index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile-main</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <div>
+     <div class="logo">
+       <h3>My Logo</h3>
+       <img class="img" src="Union.png" alt="logo of vector">
+       <img class="img3" src="Ellipse11.png" alt="Image of Ellipse">
+     </div>
+     <img class="img4" src="Rectangle4.png" alt="Image of allipse">
+     <img class="img2" src="Ellipse3.png" alt="Image of ellipse">
+     <h1>I’m Roshan <br> Glad to see <br>you!</h1>
+     <p class="para">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+     <h2>LET’S CONNECT</h2>
+     <div class="icon">
+     <img class="log2" src="Vector-5.png" alt="Logo of social media">
+     <img class="log3" src="Vector-3.png" alt="Logo of social media">
+     <img class="log3" src="Vector-2.png" alt="logo of social media">
+     <img class="log" src="Vector-1.png" alt="logo of social media">
+     <img class="log" src="Vector-4.png" alt="logo of social media">
+     <img class="log" src="Vector.png" alt="logo of social media">
+    </div>
```

Comment:
The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Please correct them.